lib/srfi/43.stk

```diff
@@ -0,0 +1,400 @@
+
+;; Incompatible procedures, which are taken from the SRFI-43
+;; reference implementation:
+;;
+;; * vector-map
+;; * vector-map!
+;; * vector-for-each
+;; * vector-fold
+;; * vector-fold-right
+;; * vector-count
+
+(define-module srfi/43
+  (import (prefix (srfi 133) s133-))
+  (export
+   ;;; Constructors
+   make-vector vector
+   vector-unfold                   vector-unfold-right
+   vector-copy                     vector-reverse-copy
+   vector-append                   vector-concatenate
+
+   ;;; Predicates
+   vector?
+   vector-empty?
+   vector=
+   
+   ;;; Selectors
+   vector-ref
+   vector-length
+
+   ;;; Iteration
+   vector-fold                     vector-fold-right
+   vector-map                      vector-map!
+   vector-for-each
+   vector-count
+
+   ;;; Searching
+   vector-index                    vector-skip
+   vector-index-right              vector-skip-right
+   vector-binary-search
+   vector-any                      vector-every
+   
+   ;;; Mutators
+   vector-set!
+   vector-swap!
+   vector-fill!
+   vector-reverse!
+   vector-copy!                    vector-reverse-copy!
+   vector-reverse!
```

Comment:
By curiosity, why is `vector-reverse!` present twice?

Reply:
> By curiosity, why is `vector-reverse!` present twice?

Hm, that's because the PR author (me) didn't notice :)
I am unable to fix it now, but will do  tomorrow.

Reply:
Fixed now :)